nibe/connection/nibegw.py

```diff
@@ -305,6 +381,37 @@ def _build(self, obj, stream, context, path):
     "_unknown" / Bytes(1), "version" / Int16ub, "model" / GreedyString("ASCII")
 )
 
+RmuData = Struct(
+    "bt1_outdoor_temperature" / Int16sl,
+    "bt7_hw_top" / Int16ul,
+    "room_sensor_setpoint_or_curve_offset_s1" / Int8ub,
```

Comment:
I tried to figure out how to adjust this field based on a future field in the flags block later, but i was unable to understand how to do that in construct.